editor/app/src/main/java/ca/ubc/eml/soiltopargraphy/editor/ui/infopanel/ImagePanelViewModel.kt

```diff
@@ -4,6 +4,6 @@ import android.arch.lifecycle.ViewModel
 import android.net.Uri
 
 class ImagePanelViewModel : ViewModel(){
-    var uri: Uri?=null
-    var imageTitle: String?=null
+    lateinit var uri : Uri
+    lateinit var imageTitle: String
```

Comment:
Don't go too overboard with `lateinit`, it makes sense for ViewModels since they are initialized once, but for properties inside the viewmodel that are set by the user they should be left as nullable.